src/stories/ReviewCard.stories.tsx

```diff
@@ -63,10 +65,24 @@ export const Basic: Story = {
 
 /** When a review has a comment but is unpublished (only the author can see this). */
 export const UnpublishedComment: Story = {
-  args: { ...defaultArgs, review: { ...review, isPublished: false } },
+  args: {
+    ...defaultArgs,
+    review: {
+      ...review,
+      comment: {
+        ...review.comment,
+        body: review.comment?.body || '',
+        status: 'unpublished',
+        type: 'Comment',
```

Comment:
I was getting type errors without the `body` and `type` provided here.

Reply:
Maybe declare the comment as its own typed object above the review:
```javascript
const defaultComment: Comment = {
  body: 'This is an example comment. Additional comments can contain harmful information so we have introduced a moderation process to protect the community.',
  status: 'published',
  type: 'Comment',
}
...
  // in the review object
  comment: defaultComment
...
// in the UnpublishedComment story:
review: {
      ...review,
      comment: {
        ...defaultComment,
        status: 'unpublished',
      },
    },
```
Dont forget to import the Comment type. There's a built-in Comment type that's not what we want and will throw errors.